spec/features/take_one_time_shift.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe "Volunteer takes One-Time Shifts" do
+  let(:log) { create(:log, when: "Tue, 31 Jan 2017") }
+  let(:volunteer) { create(:volunteer, :not_waived, regions: [log.region], assigned: true) }
+  let(:location) { create(:location) }
+  #Not sure how 'Recipient(s)' works, location.log_recipients returns ActiveRecord collection, location.recip_category returns 'nil'. What should the log show for this attribute?
```

Comment:
A Log has one "donor" and many "recipients". So if a volunteer is supposed to pick up food from King Soopers and take some of that food to Food Pantry A and the rest of the food to Food Pantry B, the Log for that would have one Donor ("King Soopers") and two Recipients ("Food Pantry A" and "Food Pantry B").

Each row in the table at `/logs/open` should show the `name` attributes of the Donor and all of the Recipients.

For now, it'd be fine to only test a log with a _single_ Recipient, but it would probably be good to explicitly specify the Donor and the Recipient for your created Log because of [the random behavior in the Log factory](https://github.com/boulder-food-rescue/food-rescue-robot/blob/take_one_time_shift_spec/spec/factories/logs.rb#L7-L19).

Something like:

```ruby
let(:donor) { create(:donor) }
let(:recipients) { create(:recipient) }
let(:log) { create(:log, donor: donor, recipients: [recipient] }
```

and then assert that the view shows the `name` of the log's donor and the `name` of its recipient:

```ruby
expect(page).to have_content(donor.name)
expect(page).to have_content(recipient.name)
```

Does that make sense?